### PR TITLE
modify doctor and patient

### PR DESCRIPTION
**Pull request by futuretech6**

### CODE REVIEW

File: pkg/account/handler.go [hunk 1]
```diff
@@ -103,6 +113,60 @@ func (h *AccountHandler) CreateAccount(c echo.Context) error {
...
 }
 
+// @Summary set doctor info
+// @Description
+// @Tags Account
+// @Produce json
+// @Param Department path string true "doctor department"
+// @Success 200 {string} api.ReturnedData{data=nil}
+// @Failure 400 {string} api.ReturnedData{data=nil}
+// @Router /account/create [POST]
```
Comment (AsterNighT) [inline pkg/account/handler.go]:
```suggestion
// @Router /account/setdoctor [POST]
```

besides, set doctor is an awful name. What does it set?

File: pkg/account/handler.go [hunk 2]
```diff
@@ -103,6 +113,60 @@ func (h *AccountHandler) CreateAccount(c echo.Context) error {
...
+}
+
+// @Summary set patient info
+// @Description
+// @Tags Account
+// @Produce json
+// @Param Allergy path string true "patient allergy history"
+// @Success 200 {string} api.ReturnedData{data=nil}
+// @Failure 400 {string} api.ReturnedData{data=nil}
+// @Router /account/create [POST]
```
Comment (AsterNighT) [inline pkg/account/handler.go]:
```suggestion
// @Router /account/setpatient [POST]
```

awful name too

**Review by AsterNighT — CHANGES_REQUESTED** [review]

**Review by AsterNighT — APPROVED** [review]

**Review by AsterNighT — CHANGES_REQUESTED** [review]
Uhh, it cannot build

**Review by RupertPaoZ — COMMENTED** [review]
we need a birthday attr in patient or account

**Comment by RupertPaoZ** [comment]
and sex

**Comment by RupertPaoZ** [comment]
and gender

**Review by AsterNighT — APPROVED** [review]

**Pull request merged**